packages/event-handler/src/rest/utils.ts

```diff
@@ -105,3 +108,39 @@ export const isAPIGatewayProxyResult = (
       typeof result.isBase64Encoded === 'boolean')
   );
 };
+
+export const composeMiddleware = (middlewares: Middleware[]): Middleware => {
```

Comment:
This function could use a bit of documentation, it's not immediately clear what it does, but I am ok for you to add it in one of the next PRs.

Reply:
Oh yeah sorry meant to add that. Will do it now.